src/js/events/CosmosPackagesActions.js

```diff
@@ -183,7 +183,10 @@ const CosmosPackagesActions = {
       },
       method: "POST",
       url: `${Config.rootUrl}${Config.cosmosAPIPrefix}/describe`,
-      data: JSON.stringify({ packageName, packageVersion }),
+      data: JSON.stringify({
+        packageName,
+        ...(packageVersion && { packageVersion })
```

Comment:
this only adds the `packageVersion`-property if it is truthy, which is what we are looking for here.

i like this syntax and think it's concise, if you feel like it is confusing, we could add everything and filter unset stuff out, e.g. by introducing a dependency on our `Util`:

```
import Util from "../../utils/Util"

// ....
  data: Json.stringify(Util.filterEmptyValues({ packageName, packageVersion }))
// ...
```

Reply:
Syntax is not confusing after the initial read